src/main/java/uk/gov/hmcts/reform/blobrouter/config/SwaggerConfiguration.java

```diff
@@ -17,7 +19,8 @@ public Docket api() {
         return new Docket(DocumentationType.SWAGGER_2)
             .useDefaultResponseMessages(false)
             .select()
-            .apis(RequestHandlerSelectors.any())
+            .apis(basePackage(Application.class.getPackage().getName() + ".controllers")
+                      .or(basePackage(Application.class.getPackage().getName() + ".reconciliation.controller")))
```

Comment:
Can you double check the outcome and (maybe) take a small screenshot of those APIs produced by this config?

Reply:
If all works, we could `Application.class.getPackage().getName()` move to some static value as well

Reply:
![image](https://user-images.githubusercontent.com/3225439/91315229-56d03480-e7af-11ea-8183-90680327f583.png)